### PR TITLE
Update notebooks submodule and add tutorial view to docs

### LINKED ISSUE #9560: Add tutorial view to docs
## 📚 Documentation

Add the tutorial view from [here](https://pytorchlightning.github.io/lightning-tutorials/) into the PL docs.

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

`git submodule update --remote`

Fixes #9560

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
@ethanwharris mind add a page with the new notebooks?
cc: @aribornstein

**Review by SeanNaren — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGMT !

File: docs/source/index.rst [hunk 1]
```diff
@@ -3,10 +3,16 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-PyTorch Lightning Documentation
-===============================
+PyTorch Lightning
+=================
```
Comment (awaelchli) [inline docs/source/index.rst:7]: Let's also add a proper introduction to the front page in a follow up? I can help :) 

Right now it starts with tutorials and the first thing that shows up is an introduction to PyTorch which is not right.

**Review by awaelchli — APPROVED** [review]

File: docs/source/index.rst (hunk 1 above)
Comment (awaelchli) [reply docs/source/index.rst:7]: @tchaton  :)

Comment (ethanwharris) [reply docs/source/index.rst:7]: Yeah, would be good to do this in a follow-up :smiley:

File: docs/source/index.rst [hunk 2]
```diff
@@ -85,19 +91,13 @@ PyTorch Lightning Documentation
...
    From PyTorch to PyTorch Lightning [Video] <https://www.youtube.com/watch?v=QHww1JH7IDU>
-   notebooks/lightning_examples/mnist-hello-world.ipynb
-   notebooks/lightning_examples/datamodules.ipynb
-   notebooks/lightning_examples/cifar10-baseline.ipynb
-   notebooks/lightning_examples/basic-gan.ipynb
-   notebooks/lightning_examples/mnist-tpu-training.ipynb
-   notebooks/lightning_examples/text-transformers.ipynb
-   notebooks/lightning_examples/reinforce-learning-DQN.ipynb
-   notebooks/lightning_examples/augmentation_kornia.ipynb
+   notebooks/**/*
```
Comment (Borda) [inline docs/source/index.rst:100]: lest keep selection here :]

**Pull request merged**